src/utils/config.ts

```diff
@@ -56,43 +47,14 @@ export const TEST = ENVIRONMENT === Environment.test;
 
 export const APP_VERSION = process.env.APP_VERSION;
 
-const DEFAULT_HOST = 'http://localhost:3000';
-
-export const BUILDER_HOST = {
-  name: Context.Builder,
-  url: new URL(process.env.BUILDER_CLIENT_HOST ?? DEFAULT_HOST),
-};
-
-export const PLAYER_HOST = {
-  name: Context.Player,
-  url: new URL(process.env.PLAYER_CLIENT_HOST ?? DEFAULT_HOST),
-};
-
-export const LIBRARY_HOST = {
-  name: Context.Library,
-  url: new URL(process.env.LIBRARY_CLIENT_HOST ?? DEFAULT_HOST),
-};
-
-export const ACCOUNT_HOST = {
-  name: Context.Account,
-  url: new URL(process.env.ACCOUNT_CLIENT_HOST ?? DEFAULT_HOST),
-};
+export const CLIENT_HOST = process.env.CLIENT_HOST ?? 'http://localhost:3000';
```

Comment:
I am not sure about the default for this ... should it not be `http://localhost:3114` or whatever we decide to run the front on ?